src/applications/Machine.java

```diff
@@ -39,35 +39,35 @@ public int getNumTasks() {
         return numTasks;
     }
 
-    public void setNumTasks(int numTasks) {
-        this.numTasks = numTasks;
-    }
-
     public Job getActiveJob() {
         return activeJob;
     }
 
-    public boolean isInactive() {
-        return (activeJob == null);
-    }
-
-    public boolean noJobQueued() {
-        return jobQ.isEmpty();
+    public boolean isActive() {
```

Comment:
Did you just like the positive voice here? Or did this make things more readable when it's called?

Reply:
We noticed the only change happening purely based on a machine being inactive was setting lastJob to null. So we though, if just initialized lastJob(activeJob after we renamed it) to be null and overwrite it's value after checking the exact opposite, if machine is active, we can get rid of the if-else statements.  Changing the function to be isActive() seemed better for readibility than doing !isInactive().